ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -201,7 +201,22 @@ tests:
   optional: true
   steps:
     env:
-      COLLECTION_METHOD: core_bpf
+      COLLECTION_METHOD: ebpf
+    workflow: stackrox-automation-flavors-powervs-e2e
+  timeout: 5h0m0s
+- as: powervs-4-14-qa-e2e-tests
```

Comment:
@msugakov @gavin-stackrox Is it fine to add separate jobs for different ocp versions?

Reply:
@mdafsanhossain yes